server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -267,6 +287,105 @@ TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) thro
         throw new Exception("Handler not present for the provided request");
     }
 
+    /**
+     * Iterates through all discovered extensions, sends transport requests for named writeables and consolidates all entires into a central named writeable registry for extensions.
+     */
+    public void getNamedWriteables() {
```

Comment:
Similar comment as the SDK. 
Can we fold all of this logic to Another Class under extensions package?